File: app/src/main/java/ly/pp/justpiano3/entity/SimpleUser.java
package ly.pp.justpiano3.entity;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.Date;

/**
 * @author as
 */
@Data
@AllArgsConstructor
public class SimpleUser {

    /**
     * 用户性别
     */
    private String gender;

    /**
     * 用户名称
     */
    private String name;

    /**
     * 用户等级
     */
    private Integer lv;

    /**
     * 某些时间(临时时间存放变量)
     */
    private Date date;

    public SimpleUser(String gender, String name, Integer lv) {
        this.gender = gender;
        this.name = name;
        this.lv = lv;
    }
}

File: app/src/main/java/ly/pp/justpiano3/entity/SimpleUser.kt
package ly.pp.justpiano3.entity

import java.util.*

/**
 * @author as
 */
data class SimpleUser(
    /**
     * 用户性别
     */
    var gender: String,

    /**
     * 用户名称
     */
    var name: String,

    /**
     * 用户等级
     */
    var lv: Int,

    /**
     * 某些时间(临时时间存放变量)
     */
    var date: Date,
)